packages/fxa-settings/src/components/Settings/ConnectedServices/index.tsx

```diff
@@ -32,10 +32,12 @@ export function sortAndFilterConnectedClients(
   // get a unique (by name) list and sort by time last accessed
   const sortedAndUniqueClients = Object.keys(groupedByName)
     .map((key) => {
-      return groupedByName[key].sort(
-        (a: AttachedClient, b: AttachedClient) =>
-          a.lastAccessTime - b.lastAccessTime
-      )[0];
+      return groupedByName[key]
+        .sort(
+          (a: AttachedClient, b: AttachedClient) =>
+            a.lastAccessTime - b.lastAccessTime
+        )
+        .reverse()[0];
```

Comment:
I think I'm confused&hellip; So we sort by `.lastAccessTime` (I think in _ascending_ order if my brain is doing that right), then we _`reverse()`_ into **descending** order and pluck the first value?
Isn't this the same as reversing the variables to `b.lastAccessTime - a.lastAccessTime` and then avoiding the `reverse()`?

Looking at the `AttachedClient` interface, `lastAccessTime` is a number, so not sure if you could even abuse `Math.max(a.lastAccessTime, b.lastAccessTime)`, but now that I think about it, probably not since `.sort()` likes -1 // 0 // +1 values.

Reply:
No, I don't think you are confused.

And thanks to your comment I noticed that there's another `sort` call involving `lastAccessTime` values on line 42 below.  😖  I will look at it with clearer eyes tomorrow.

Reply:
I'm permanently confused. 
Was trying to think if this could be smashed down to a single `.reduce()`, but still think it'd need `Object.keys()` (or `Object.values()`) and at least 1 `.sort()` so it started feeling like a bit of a wash.

Reply:
> Isn't this the same as reversing the variables to b.lastAccessTime - a.lastAccessTime and then avoiding the reverse()?

Yeah, that should work.  I'll updated the PR.